### ISSUE TITLE
[A11Y] [iOS] Unable to linear navigate to the items in Preview banner.

### ISSUE THREAD

**Issue by spotukuchi** [ampproject/ampstart#408]
ENVIRONMENT:
Form factor : iPAD
Model : MC705LL/A
Version : 9.3.5(13G36)
Browser : Safari

PRE-REQUISITE:
Turn 'VoiceOver' ON. ( Settings->General->Accessibility->VoiceOver->ON )

STEPS TO REPRODUCE:
1. Navigate to the URL: https://ampstart.com/
2. Navigate to "Templates" link and double tap on it.
3. Navigate to 'VIEW TEMPLATE' in any of the sections and double tap on it.
4. Try to linear navigate through all the items in the Preview banner.

ACTUAL RESULTS:
Focus does not move onto the Preview banner through linear navigation. Focus is on the background text content. Only exploratory touch works.

EXPECTED RESULTS:
User should be able to navigate through all the items in the preview banner through linear navigation.

ATTACHMENT:
Please see the video in the link below for reference.

https://drive.google.com/corp/drive/folders/0BwgKOOZP99EuUzVWT25TVzl2S00

**Comment by torch2424** [comment]
Duplicate of #407

Please see comment on that issue as well if interested
